### PR TITLE
Initial support to Brazilian Portuguese (pt-br)

### PR DESCRIPTION
**Pull request by dyorgio**
#### Database Migration
NO

#### Description
Included pt-br codes and adjust IsoLanguage. Find to work with any lenght code

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR

* https://github.com/Radarr/Radarr/issues/1626

### CODE REVIEW

File: src/NzbDrone.Core/Parser/IsoLanguages.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ public static class IsoLanguages
                                                                new IsoLanguage("fi", "fin", Language.Finnish),
                                                                new IsoLanguage("tr", "tur", Language.Turkish),
                                                                new IsoLanguage("pt", "por", Language.Portuguese),
+                                                               new IsoLanguage(new List<string> {"pt-br", "pt-BR"}, "pob", Language.Brazilian),
```
Comment (Qstick) [inline src/NzbDrone.Core/Parser/IsoLanguages.cs]: This should just be the two digit code, br

Comment (dyorgio) [reply src/NzbDrone.Core/Parser/IsoLanguages.cs]: Two digit code br is for another language (Breton).
Brazilian Portuguese oficial iso code is pt-br (variant of pt (Portuguese of Portugal)

**Comment by Osodarck** [comment]
@Qstick @dyorgio Is there something I can do to help this to be commited?

**Comment by dyorgio** [comment]
> @Qstick @dyorgio Is there something I can do to help this to be commited?

Hi @Osodarck , I fix new conflics. I can't help you, I'm trying to include Brazillian portuguese support 3 times, the final decision is always of Radarr team.

**Comment by pauloloks** [comment]
Conseguiu a tradução ?

**Comment by juanmoura** [comment]
Any new with review guys?

**Comment by rflaperuta** [comment]
It would be nice to have pt-br in Radarr.... If time allows, can someone please take a look at it?

**Comment by marcioamoke** [comment]
please,  include pt-br as a new language.

**Comment by luisandre-debug** [comment]
please, include pt-br as a new language😪😪😪

**Comment by lucaspinheiroxp** [comment]
please, include pt-br as a new language.

**Comment by Qstick** [comment]
please, include USEFUL comments about pt-br as a new language, or this PR will be closed. 

If you want to spam advocating the feature do it elsewhere.

**Pull request closed without merging**